.github/workflows/build-packages.yml

```
@@ -19,6 +22,14 @@ jobs:
         with:
           name: package-parts-linuxmusl-x86-64
           path: agent/native/_build/linuxmusl-x86-64-release/
+
+      - name: Log in to the Container registry
+        uses: docker/login-action@e92390c5fb421da1463c202d546fed0ec5c39f20
```

IIRC, dependabot works with the explicit version in a comment too

```suggestion
        uses: docker/login-action@e92390c5fb421da1463c202d546fed0ec5c39f20  # v3.1.0
```

---

Ref: [714919b](https://github.com/elastic/apm-agent-php/pull/1145/commits/714919bdf86d9a979d0f96d22a90343d2865256a)